cmd/src/actions_exec.go

```diff
@@ -540,38 +538,44 @@ fragment repositoryFields on Repository {
 		} `json:"errors,omitempty"`
 	}
 
-	if err := (&apiRequest{
-		query: query,
-		vars: map[string]interface{}{
-			"query": scopeQuery,
-		},
-		// Do not unpack errors and return error. Instead we want to go through
-		// the results and check whether they're complete.
-		// If we don't do this and the query returns an error for _one_
-		// repository because that is still cloning, we don't get any repositories.
-		// Instead we simply want to skip those repositories that are still
-		// being cloned.
-		dontUnpackErrors: true,
-		result:           &result,
-	}).do(); err != nil {
-
-		// Ignore exitCodeError with error == nil, because we explicitly set
-		// dontUnpackErrors, which can lead to an empty exitCodeErr being
-		// returned.
-		exitCodeErr, ok := err.(*exitCodeError)
-		if !ok {
-			return nil, err
-		}
-		if exitCodeErr.error != nil {
-			return nil, exitCodeErr
+	// Helper function to perform a graphql request
+	doRequest := func(query string, vars map[string]interface{}, dest interface{}) error {
+		if err := (&apiRequest{
+			query: query,
+			vars: map[string]interface{}{
+				"query": scopeQuery,
```

Comment:
The `vars` you pass in to `doRequest` are not used. Instead you always pass in this set of `vars`.